Netable/Example/ViewModels/UserVM.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UserVM.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-12-01.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Foundation
+
+class UserVM: ObservableObject {
+    @Published var user: User?
+
+
```

Comment:
It might be worth either adding swiftlint to this project temporarily before we merge - I've noticed a handful of these little polish-y typos